ecs-cli/modules/cli/cluster/cluster_app.go

```diff
@@ -192,20 +192,25 @@ func createCluster(context *cli.Context, ecsClient ecsclient.ECSClient, cfnClien
 
 	}
 
+	// Check if cluster is specified
+	if commandConfig.Cluster == "" {
+		return fmt.Errorf("Please configure a cluster using the configure command or the '--%s' flag", flags.ClusterFlag)
```

Comment:
w/o this check, what happens if a cluster isn't specified? does it always error out later, or are there circumstances when cluster's not required (like pushing ECR images) ?